### PR TITLE
docs: add an example of variable swapping

### PR DESCRIPTION
**Pull request by EPonyA**
Wrote an example of how multiple variable assignment can be used to swap two variables.

### CODE REVIEW

**Comment by EPonyA** [comment]
Noob question, but why does this seem to have a formatting error? I get this output in my terminal:
```
➜  v git:(docs_val_swap) ✗ v fmt tmp.v
mut a := 0
mut b := 1
println('$a, $b') // 0, 1
a, b = b, a
println('$a, $b') // 1, 0
```

**Comment by EPonyA** [comment]
I recognized the problem. My bad!

File: doc/docs.md [hunk 1]
```diff
@@ -326,6 +326,16 @@ fn main() {
 }
 ```
 
+The values of multiple variables can be changed in one line. In this way, their values can be swapped without an intermediary variable.
```
Comment (spytheman) [inline doc/docs.md:329]: This line is too long and should be wrapped. Perhaps after the first sentence?

Comment (EPonyA) [reply doc/docs.md]: Thank you! It seems to pass now. Are there any other problems?

Comment (spytheman) [reply doc/docs.md]: no, everything seems fine

**Pull request merged**